corehq/form_processor/submission_post.py

```diff
@@ -84,6 +86,7 @@ def _set_submission_properties(self, xform):
         xform.app_id = self.app_id
         xform.build_id = self.build_id
         xform.export_tag = ["domain", "xmlns"]
+        xform.partial_submission = self.partial_submission
```

Comment:
@snopoke it seemed like this is the only place i had to add this, and the tests that rely on this pass with it. can you confirm this is all that needs to be done?